cmd/kubeadm/app/constants/constants.go

```diff
@@ -83,3 +78,8 @@ const (
 	// The file name of the tokens file that can be used for bootstrapping
 	CSVTokenFileName = "tokens.csv"
 )
+
+var (
+	AuthorizationPolicyPath        = path.Join(KubernetesDir, "abac_policy.json")
+	AuthorizationWebhookConfigPath = path.Join(KubernetesDir, "webhook_authz.conf")
```

Comment:
It isn't new. We decided on making these params hard coded -- at least until we have the full config file backend to rely on where the user can specify params that are passed directly to the apiserver/cm/scheduler